src/index.ts

```diff
@@ -6,7 +6,7 @@ import { validateTx } from './txValidators'
 import type { RawTransaction, SignedTransaction, TransactionBody } from './types'
 import { decodeCbor, encodeToCbor } from './utils'
 
-export type { RawTransaction, SignedTransaction, TransactionBody }
+export * from './types'
```

Comment:
We are using many of these types in hw-cli, so we need to export them.

Is it better to explicitly list all types we want to export here (most of them) or is the wildcard okay?